docs/quickstart.rst

```diff
@@ -24,6 +24,31 @@ You will see a Studio project similar to the image below. The Studio project pro
 
 .. image:: img/studio_project_overview.png
 
+1.5 Set up a Virtual environment (Recommended)
+----------------------------------------------
+
+Before installing FEDn using pip, we recommend creating a virtual environment. This helps isolate dependencies and avoids conflicts with other Python projects on your machine.
+
+You can set up and activate a virtual environment using the following steps:
+
+**Using venv** (Python's built-in module for virtual environments)
+
+.. tabs::
+
+    .. code-tab:: bash
+         :caption: Unix/MacOS
+
+         python3 -m venv fedn_env
+         source fedn_env/bin/activate
+
+    .. code-tab:: bash
+         :caption: Windows (Powershell)
+
+         python -m venv fedn_env
+         .\fedn_env\Scripts\Activate
```

Comment:
have you tested this? I was not able to do this is my powershell because of admin rights, instead I used CMD.exe and then there is a .bat file to activate instead.

Reply:
I'll do further testing later today on my windows machine. 
Wouldn't switching to CMD be a bad decision since we later use PowerShell to start the clients?
A workaround could also be starting PowerShell in admin mode?

Reply:
I've tested it and it works on my windows machine, I've added a line to bypass the admin permissions if that error shows up